pom.xml

```diff
@@ -51,13 +51,19 @@
         <!-- temporary fix by pinning the version until we upgrade to a version of common that contains this or newer version.
             See https://github.com/confluentinc/common/pull/332 for details -->
         <dependency.check.version>6.1.6</dependency.check.version>
+        <aws-java-sdk.version>1.11.1029</aws-java-sdk.version>
     </properties>
 
     <repositories>
         <repository>
             <id>confluent</id>
             <name>Confluent</name>
-            <url>https://packages.confluent.io/maven/</url>
```

Comment:
Remove this line

Reply:
AWS interceptor is not available in packages.confluent.io/maven reposiotry